### PR TITLE
TSK-1236: Remove links from the export of classification

### PR DESCRIPTION
**Pull request by gitgoodjhe**
---
 
### For the submitter:
- [ ] I updated the [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview) and will supply links to the specific files
- [ ] I did not update the [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview)
- [x] I included a link to the [SonarCloud branch analysis](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1019969636/SonarCloud+Integration)
- [ ] I added a description of changes on the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana)
- [x] I did not update the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana)
- [x] I put the ticket in review
- [ ] After integration of the pull request, I verified our [bluemix test environment](http://taskana.mybluemix.net/taskana) is not broken

### Verified by the reviewer:
- [ ] Commit message format → TSK-XXX: Your commit message.
- [ ] Submitter's update to [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview) is sufficient
- [ ] SonarCloud analysis meets our standards
- [ ] Update of the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana) reflects changes
- [ ] PR fulfills the ticket
- [ ] Edge cases and unwanted side effects are tested
- [ ] Readability

### CODE REVIEW

**Comment by gitgoodjhe** [comment]
https://sonarcloud.io/dashboard?branch=TSK-1236&id=gitgoodjhe_taskana

File: rest/taskana-rest-spring/src/main/java/pro/taskana/classification/rest/ClassificationDefinitionController.java [hunk 1]
```diff
@@ -84,6 +84,10 @@
                 Collectors.collectingAndThen(
                     Collectors.toList(), classificationRepresentationModelAssembler::toPageModel));
 
+    pageModel
+        .getContent()
+        .forEach(ClassificationRepresentationModel::removeLinks);
```
Comment (holgerhagen) [inline rest/taskana-rest-spring/src/main/java/pro/taskana/classification/rest/ClassificationDefinitionController.java:89]: I would rather expect that we do remove the creation of the links?

**Review by holgerhagen — CHANGES_REQUESTED** [review]

File: rest/taskana-rest-spring/src/main/java/pro/taskana/classification/rest/ClassificationDefinitionController.java (hunk 1 above)
Comment (mustaphazorgati) [reply rest/taskana-rest-spring/src/main/java/pro/taskana/classification/rest/ClassificationDefinitionController.java:89]: Me too

**Comment by gitgoodjhe** [comment]
Will be included in Mustaphas REST-refactoring PR

**Pull request closed without merging**